toolkit/tools/pkg/imagecustomizerlib/configvalidation.go

```diff
@@ -54,6 +54,8 @@ func ValidateConfig(ctx context.Context, baseConfigPath string, config *imagecus
 		Options:        options,
 	}
 
+	ResolveBaseConfigs(ctx, rc)
```

Comment:
I would probably move this below the `config.IsValid()` and `options.IsValid()` checks, just so you know the config has basic validity before trying to merge the base configs.